guide/additional-features/cooldowns.md

```diff
@@ -0,0 +1,87 @@
+# Cooldowns
+
+Spam is something you generally want to avoid, especially if one of commands require calls to other APIs or takes a bit of time to build/send.
+
+This section assumes you followed the [Command Handling](/guide/creating-your-bot/command-handling.md) part.
+
+First, add a cooldown property to your command. This will determine how long the user would have to wait (in seconds) before using the command again.
+
+```js {4}
+const { SlashCommandBuilder } = require('discord.js');
+module.exports = {
+	cooldown: 5,
+	data: new SlashCommandBuilder()
+		.setName('ping')
+		.setDescription('Replies with Pong!'),
+	async execute(interaction) {
+		// ...
+	},
+};
+```
+
+In your main file, initialize a [Collection](/guide/additional-info//collections.md) to store cooldowns of commands
+
+```js {1}
```

Comment:
I don't think the highlight is needed, especially if there is only one line of code.